Marlin/src/gcode/motion/G2_G3.cpp

```diff
@@ -362,7 +369,7 @@ void plan_arc(
       #endif
 
       // calculate safe speed for stopping by the end of the arc
-      arc_mm_remaining -= segment_mm;
+      const float arc_mm_remaining = flat_mm - segment_mm * i;
```

Comment:
As above, is multiplication actually more accurate than successive subtractions?